src/_common/popper/AppPopper.vue

```diff
@@ -299,6 +312,8 @@ onMounted(() => {
 			}
 		},
 	});
+
+	onManualShow();
```

Comment:
this might not be how we're used to doing things. the watcher didnt seem to trigger when the component loaded so even if you set trigger=manual and manual-show it wouldnt show the popper when it loads in.

is there a better way to trigger this to happen?